packages/resolve-scripts/src/server/auth/helpers.js

```diff
@@ -0,0 +1,44 @@
+import cookie from 'cookie'
+
+const createRequest = expressReq => ({
+  path: expressReq.path,
+  method: expressReq.method,
+  originalUrl: expressReq.originalUrl,
+  headers: expressReq.headers,
+  params: expressReq.params,
+  body: expressReq.body,
+  query: expressReq.query,
+  cookies: expressReq.cookies,
+  resolve: expressReq.resolve
+})
+
+const createResponse = expressRes => {
+  const response = {
+    expressRes,
+    clearCookie: name => {
+      const fieldContentRegExp = /^[\u0009\u0020-\u007e\u0080-\u00ff]+$/
+
+      if (!fieldContentRegExp.test(name)) {
+        throw new TypeError('argument name is invalid')
+      }
+
+      response.cookie(name, 'deleted', {
+        expires: new Date(0)
+      })
+    },
+    cookie: (name, value, options) => {
+      expressRes.append('Set-Cookie', cookie.serialize(name, value, options))
+    }
+  }
+
+  return response
+}
+
+const getRouteByName = (name, routes) => {
+  const route = routes[name]
+  const { path = route, method = 'get' } = route
+  if (typeof path !== 'string') return null
```

Comment:
What is the target of this check? If you want to know, that 'path' exists, you should use (!path) or (path === undefined) or something like this. If you want to check path validation, you should check exactly validation: format and type of your path, and use regex operators. If any view of 'path' is allowed just change its type to string (for example, .toString() method)

Reply:
This check is correct, we support only string, we don't support object, function or others

Reply:
If your path = "something", it's not correct too, right? Maybe we should check the type of your string, that it has the "/" symbol?

Reply:
In this case your application just do nothing, maybe it must to output clear error for developer.